core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -260,4 +260,12 @@
             margin-bottom: $spacing-3;
         }
     }
+
+    tiptap-editor {
+        /* To avoid jupms whe the scroll is hidden */
```

Comment:
Small typo in the comment "jumps" @rjvelazco

Reply:
Thanks, @manuelrojas!